<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MSI" failures="1" tests="2" name="com.parabankautomation.testcases.LoginTest" time="28.919" errors="0" timestamp="2025-07-04T10:30:12 IST" skipped="0">
  <testcase classname="com.parabankautomation.testcases.LoginTest" name="TC002_LoginWithValidusernameInvalidPassword" time="14.011"/>
  <testcase classname="com.parabankautomation.testcases.LoginTest" name="TC001_LoginWithValidCredentials" time="14.908">
    <failure message="expected [ParaBank | Accounts Overview] but found [ParaBank | Error]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [ParaBank | Accounts Overview] but found [ParaBank | Error]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.parabankautomation.helpers.UiHelpers.CompareStrings(UiHelpers.java:8)
at com.parabankautomation.pages.LoginPage.LoginWithValidCredentialsAndVerifyTitle(LoginPage.java:71)
at com.parabankautomation.testcases.LoginTest.TC001_LoginWithValidCredentials(LoginTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- TC001_LoginWithValidCredentials -->
</testsuite> <!-- com.parabankautomation.testcases.LoginTest -->
